di_website/visualisation/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.shortcuts import redirect
+
+from wagtail.core.models import Page
+from wagtail.core.fields import StreamField
+from wagtail.admin.edit_handlers import StreamFieldPanel
+
+from .blocks import ChartBlock
+
+
+class VisualisationsPage(Page):
+    """
+    Parent page for all visualisations
+    """
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['visualisation.ChartPage']
+
+    class Meta:
+        verbose_name = 'Visualisations Page'
+
+    def get_sitemap_urls(self, request):
+        return []
+
+    def serve(self, request, *args, **kwargs):
+        return redirect(self.get_parent().url)
+
+    def serve_preview(self, request, mode_name):
+        return self.get_parent().serve(request)
+
+
+class ChartPage(Page):
+    parent_page_types = [VisualisationsPage]
+    subpage_types = []
+
+    charts = StreamField([('chart', ChartBlock())], blank=True)
+
+    content_panels = Page.content_panels + [
+        StreamFieldPanel('charts')
+    ]
+
+    class Meta:
+        verbose_name = 'Chart Page'
+
+    def get_sitemap_urls(self, request):
+        return []
+
+    def serve(self, request, *args, **kwargs):
```

Comment:
@edwinmp 

This method needs modifying to return a valid response if the user is authenticated.

If the user is authenticated (i.e. a logged-in CMS editor), it should return a simple template that displays the chart in isolation.

 I'm guessing this should be the same HTML include that is used to render the chart elsewhere on the site, a bare-bones HTML `<head>`, and whatever supporting JS is required.

We can attack this at some point in the pattern library if that helps?

If the user is not a CMS user, this method should return a 404 as per its parent page type.